lib/test/Test/Pos/Types/BlockSpec.hs

```diff
@@ -1,5 +1,7 @@
 {-# LANGUAGE RankNTypes #-}
 
+{-# OPTIONS_GHC -fno-warn-deprecations #-}
```

Comment:
What's been deprecated?

Reply:
My bad - this was a `Debug.Trace` usage while fixing a test.